### PR TITLE
update ubuntu deps in README

### PR DESCRIPTION
**Pull request by blacktop**
I believe these are the packages you need for ubuntu as the ones you listed were not found by `apt-get`

### CODE REVIEW

**Comment by iangcarroll** [comment]
They work, at least with Debian, if you enable the correct architecture and then `apt update`: `dpkg --add-architecture i386`

**Comment by blacktop** [comment]
@iangcarroll thank you for that info!

File: README.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ $ make
 
 | Fedora / RedHat       | Ubuntu / Debian        | Comment                      |
 | --------------------- | ---------------------- |:---------------------------- |
-| `glibc-devel.i686`    | `libc6-dev:i386`       |                              |
+| `glibc-devel.i686`    | `libc6-dev-i386`       |                              |
 | `libgcc.i686`         | `gcc-multilib`         |                              |
-| `readline-devel.i686` | `libreadline-dev:i386` | Optional, used in mpscript.  |
+| `readline-devel.i686` | `lib32readline-dev`    | Optional, used in mpscript.  |
```
Comment (taviso) [inline README.md]: I'm not really sure about this. The existing ones seem like the correct deps on my test system, but I'm not a Ubuntu expert (Fedora guy). Do you know what's different on our systems?

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CFLAGS  = -O3 -march=native -ggdb3 -m32 -std=gnu99 -fshort-wchar -Wno-multichar -Iinclude -mstackrealign
+CFLAGS  = -O3 -ggdb3 -m32 -std=gnu99 -fshort-wchar -Wno-multichar -Iinclude -mstackrealign
```
Comment (taviso) [inline Makefile]: I think this is a good idea, but probably need to remove it from the other Makefiles as well, I should clean up the build :)

Comment (blacktop) [inline Makefile]: thanks @taviso I'll close this pull request.

**Pull request closed without merging**